contracts/truefi2/BorrowerRewardDistributor.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+/**
+ * @title BorrowerRewardDistributor
+ * @dev Reward borrowers for taking out loans in TRU
+ */
+
+contract BorrowerRewardDistributor {
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+
+    /**
+     * @notice Will be TRU
+     */
+    IERC20 public rewardCurrency;
+
+    /**
+     * @dev Reward rate is set as 1% by default. 2 decimals
+     * so the actual reward can be calculated as: loan * rewardRate / 10000
+     */
+    uint256 public rewardRate = 100;
+
+    /** @notice setting TRU address
+     */
+    constructor(IERC20 _rewardCurrency) public {
```

Comment:
add a function initialize(IERC20 _rewardCurrency, address _lender) instead of a constructor. Make initialize set:
rewardCurrency  = _rewardCurrency;
lender = _lender;
rewardRate = 100;